main/current reviews/65_AUTO_2_INVEST_TRIM_5_CI_7_CIS2_7_3_4_5_2_CROSS_CUATM.sql

```diff
@@ -0,0 +1,1505 @@
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.COL10 ELSE NULL END ) AS CONS_TS_COL10,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.COL11 ELSE NULL END ) AS CONS_TS_COL11,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.COL12 ELSE NULL END ) AS CONS_TS_COL12,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.COL13 ELSE NULL END ) AS MUNCA_2_COL13,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.COL14 ELSE NULL END ) AS MUNCA_2_COL14,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.COL15 ELSE NULL END ) AS MUNCA_2_COL15,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.auto_1_COL16 ELSE NULL END ) AS auto_1_COL16,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.auto_1_COL17 ELSE NULL END ) AS auto_1_COL17,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.auto_1_COL18 ELSE NULL END ) AS auto_1_COL18,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.CIS_5_CI__COL19 ELSE NULL END ) AS CIS_5_CI__COL19,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.CIS_5_CI__COL20 ELSE NULL END ) AS CIS_5_CI__COL20,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.CIS_5_CI__COL21 ELSE NULL END ) AS CIS_5_CI__COL21,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.TRIM_2_INV___COL22 ELSE NULL END ) AS TRIM_2_INV___COL22,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.TRIM_2_INV___COL23 ELSE NULL END ) AS TRIM_2_INV___COL23,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.TRIM_2_INV___COL24 ELSE NULL END ) AS TRIM_2_INV___COL24,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.TRIM_65_auto_COL25 ELSE NULL END ) AS TRIM_65_auto_COL25,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.TRIM_65_auto_COL26 ELSE NULL END ) AS TRIM_65_auto_COL26,
+SUM(CASE WHEN  C.FULL_CODE LIKE '%'||CR.CS_CUATM||'%'  THEN L.TRIM_65_auto_COL27 ELSE NULL END ) AS TRIM_65_auto_COL27
+
+FROM 
+
+(
+--1--
+SELECT 
+C.CODUL,
+C.FULL_CODE,
+COUNT (DISTINCT CASE WHEN FC.FORM  = 3 THEN  FC.CUIIO END ) AS COL1,
+NULL AS COL2,
+NULL AS COL3,
+COUNT (DISTINCT CASE WHEN FC.FORM  = 4 THEN  FC.CUIIO END ) AS COL4,
+NULL AS COL5,
+NULL AS COL6,
+NULL AS COL7,
+NULL AS COL8,
+NULL AS COL9,
+NULL AS COL10,
+NULL AS COL11,
+NULL AS COL12,
+NULL AS COL13,
+NULL AS COL14,
+NULL AS COL15,
+NULL AS auto_1_COL16,
+NULL AS auto_1_COL17,
+NULL AS auto_1_COL18,
+NULL AS CIS_5_CI__COL19,
+NULL AS CIS_5_CI__COL20,
+NULL AS CIS_5_CI__COL21,
+NULL AS TRIM_2_INV___COL22,
+NULL AS TRIM_2_INV___COL23,
+NULL AS TRIM_2_INV___COL24,
+
+NULL AS  TRIM_65_auto_COL25,
+NULL AS  TRIM_65_auto_COL26,
+NULL AS  TRIM_65_auto_COL27
+
+FROM 
+
+
+(
+SELECT     
+
+           R.CUIIO,
+           R. CUIIO_VERS,
+           FC.FORM,
+          
+           R.CUATM
+           
+      FROM (SELECT FC.CUIIO,
+                   FC.CUIIO_VERS,
```

Comment:
FROM (SELECT FC.CUIIO,
                   FC.CUIIO_VERS,
                   FC.FORM,
                   FC.FORM_VERS,
                   FC.STATUT
              FROM CIS.FORM_CUIIO  FC
                   INNER JOIN (  SELECT CUIIO, MAX (CUIIO_VERS) CUIIO_VERS
                                   FROM CIS.FORM_CUIIO
                                  WHERE FORM IN (3,4) AND CUIIO_VERS <= :pPERIOADA 
                               GROUP BY CUIIO) BB
                       ON (    BB.CUIIO = FC.CUIIO
                           AND BB.CUIIO_VERS = FC.CUIIO_VERS)
             WHERE FC.FORM IN (3,4) AND FC.STATUT <> '3') FC
           INNER JOIN CIS.RENIM R
               ON (R.CUIIO = FC.CUIIO AND R.CUIIO_VERS = FC.CUIIO_VERS)